### DISCUSSION TITLE
Logout all user session

### DISCUSSION THREAD

**Discussion by ssakone** [pocketbase/pocketbase#5368] [Q&A] [answered]
How to logout all user session he is login on new device

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase APIs are fully stateless (even the tokens are not stored in the database) and there are no "sessions" in the traditional sense. You could also check https://github.com/pocketbase/pocketbase/issues/2536 (and its [related discussion](https://github.com/pocketbase/pocketbase/discussions/2042#discussioncomment-5275124)).

If you need further help, please don't treat the discussions as llm chat and try to elaborate a little more on use case to understand the bigger picture and what actually you are trying to do.

**Reply by ganigeorgiev** [reply]
If what you are trying to do is for some security reason, note that MFA, OTP (email code) and alerts for auth from new devices (_currently a "device" is represented as simple md5 hash of the IP+useragent_) have been already implemented as part of the [ongoing refactoring](https://github.com/pocketbase/pocketbase/discussions/4355) and they will be available with PocketBase v0.23.0 release in the near future.

**Reply by ssakone** [reply]
Sorry, i will write better content in feature, but from you reply i find out this [2042](https://github.com/pocketbase/pocketbase/discussions/2042#discussioncomment-5275523) and its help me a lot thanks
